docs/source/using/mounting-host-folders.adoc

```diff
@@ -0,0 +1,122 @@
+[[mounting-host-folders]]
+= Mounting host folders
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[mount-host-folders-overview]]
+== Overview
+
+Some drivers mount a host folder within the VM so that you can share files between the VM and the host.
+These folders are not currently configurable, and are different for each driver and OS that you use.
+
+[cols=",,,",options="header",]
+|========================================
+|Driver |OS |HostFolder |VM
+|Virtualbox |Linux |/home |/hosthome
+|Virtualbox |OSX |/Users |/Users
+|Virtualbox |Windows |C://Users |/c/Users
+|VMWare Fusion |OSX |/Users |/Users
+|Xhyve |OSX |/Users |/Users
+|========================================
+
+NOTE: Host folder sharing is not implemented in the KVM and Hyper-V drivers. You can
+link:#mounting-custom-shared-folders[mount a CIFS-based shared folder] inside the VM instead.
+
+[[mounting-custom-shared-folders]]
+== Mounting custom shared folders
+
+The Minishift https://github.com/minishift/minishift-b2d-iso[Boot2Docker] ISO and the
+Minishift https://github.com/minishift/minishift-centos-iso[CentOS] ISO include `cifs-utils`
+and `fuse-sshfs`. These tools allow you to mount host folders using
+https://en.wikipedia.org/wiki/Server_Message_Block[CIFS]
+or https://en.wikipedia.org/wiki/SSHFS[SSHFS].
+
+NOTE: When you use the Boot2Docker ISO along with the VirtualBox driver, VirtualBox
+guest additions are automatically enabled and occupy the _/Users_ mountpoint as
+shown in the following examples. In this case you need to use a different mountpoint.
+
+[[cifs-folder-mount]]
+=== Mounting folders with CIFS
+
+In this example, you use CIFS-based sharing to mount `C:\Users` on a Windows host
+into the Minishift VM. On Windows 10, the `C:\Users` folder is shared by default
+and only needs locally-authenticated users.
+
+.  Find the local IP address from the same network segment as your
+Minishift instance.
++
+----
+$ Get-NetIPAddress | Format-Table`
+----
+
+.  Create a mountpoint and mount the shared folder.
++
+[source,sh]
```

Comment:
@Preeticp @hferentschik @thatdocslady Regarding code highlighting e.g. `[source, bash], [source, yaml]` I see them used in openshift-docs. As per AsciiDoctor documentation source highlighters supported.  However it is not mentioned in the guideline  https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks. 

Also about `[listing.console] ` , it is not present in in the existing openshift-docs or in the doc_guidelines. So I am not sure if this plays well with AsciiDoctor and the tool chain to convert downstream docs. I am expecting some tool chain is used to convert these docs to downstream docs. 

Can we please discuss and come to some guideline for Minishift docs so that  in future it is relatively clear when someone sends a documentation PR.